GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -516,28 +565,30 @@ public override string ToString()
             }
         }
 
-        private IEnumerable<MergeInfo> GetMergeInfo(string tfsPathBranchToCreate, string tfsPathParentBranch,
-            int firstChangesetInBranchToCreate, int lastChangesetIdToCheck)
+        private IList<MergeInfo> GetMergeInfo(string tfsPathBranchToCreate, string tfsPathParentBranch, int firstChangesetInBranchToCreate, int lastChangesetIdToCheck)
         {
             var mergedItemsToFirstChangesetInBranchToCreate = new List<MergeInfo>();
-            var merges = VersionControl
-                .TrackMerges(new int[] { firstChangesetInBranchToCreate },
+            var merges = VersionControl.TrackMerges(new int[] { firstChangesetInBranchToCreate },
                     new ItemIdentifier(tfsPathBranchToCreate),
                     new ItemIdentifier[] { new ItemIdentifier(tfsPathParentBranch), },
                     null)
                 .OrderByDescending(x => x.SourceChangeset.ChangesetId);
+
```

Comment:
That's a fair point - I'll keep it in mind in the future. Most of the whitespace changes were automatically performed by ReSharper as I went through; I'll have to keep an eye on it.